.cirrus.yml

```diff
@@ -76,6 +76,37 @@ base_template: &BASE_TEMPLATE
     - git checkout FETCH_HEAD  # Use merged changes to detect silent merge conflicts
                                # Also, the merge commit is used to lint COMMIT_RANGE="HEAD~..HEAD"
 
+worker_template: &WORKER_TEMPLATE
+  persistent_worker:
+    isolation:
+      vetu:
+        image: ghcr.io/cirruslabs/ubuntu-runner-amd64:22.04
+        user: admin
+        password: admin
+        disk_size: 50
```

Comment:
What if someone increases this to 1000 and fills our disk with it?

Reply:
Is the protection is that this is limited by vetu? So there would only be so far that changes here could have any effect.